include/nbl/asset/CCPUMeshPackerV2.h

```diff
@@ -44,11 +44,12 @@ class CCPUMeshPackerV2 final : public IMeshPacker<ICPUMeshBuffer, MDIStructType>
         }
     };
 
+    template<typename BufferType>
```

Comment:
nope, nope, create GPUMeshPackerV2. So that you might have 
```
template <typename BufferType>
class IMeshPackerV2 {....};

class CCPUMeshPackerV2 : public IMeshPackerV2<ICPUBuffer> {...};

class CGPUMeshPackerV2 : public IMeshPackerV2<IGPUBuffer> {...};
```
And obviously try to exclude as much common code as possible and keep it in the base class

Reply:
Look at IGPUVirtualTexture for an example

Reply:
Do it before further development of the example